src/pages/management/webhooks/v3.2/index.mdx

```diff
@@ -5,7 +5,8 @@ subcategory: "webhooks"
 title: "Webhooks"
 tagline: "The refrence of LiveChat webhooks."
 desc: "The refrence of available LiveChat webhooks. Use webhooks to build automated flows in your integrations."
-apiVersion: "3.2"
```

Comment:
Here, I hid the document as the document should be hidden together with all v3.2 documents, and apparently, it was missed by me while hiding all the documents